plugins/oss/pipeline-as-a-code/dinghy.yml

```diff
@@ -0,0 +1,41 @@
+autoLockPipelines: false
+dinghyFilename: dinghyfile
+dinghyIgnoreRegexp2Enabled: false
+echo:
+  baseURL: ${services.echo.baseUrl}
+enabled: true
```

Comment:
Shouldn't `enabled` be indented and set the same way other services are `'{services.echo.enabled}'`?

Reply:
No, this is a flag for Dinghy itself, whether it is enabled or not. It should be at the top level.